exchanges/btcmarkets/btcmarkets.go

```diff
@@ -483,6 +483,17 @@ func (b *BTCMarkets) RemoveOrder(ctx context.Context, id string) (CancelOrderRes
 		request.Auth)
 }
 
+// Replace removes a given order
+func (b *BTCMarkets) ReplaceAnOrder(ctx context.Context, id string, price, amount float64) (*OrderData, error) {
```

Comment:
Please add `clientOrderId` support

Reply:
How bout I also add the price and amount as well. 😆